### PR TITLE
draft v20.1.11 release notes

### PR DESCRIPTION
**Pull request by taroface**

### CODE REVIEW

**Comment by taroface** [comment]
@mikeCRL For reference, our steps for generating release notes are here (see in particular "Update the docs site"): https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-16.Updatedocs(handledbytheDocsteam)

File: releases/v20.1.11.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+~~~
+
+
+### SQL language changes
+
+- Fix a bug in [`RESTORE`](../v20.1/restore.html) where some unusual range boundaries in interleaved tables caused an error. [#58260][#58260]
+
+### Bug fixes
+
+- In v20.1.8, we attempted to fix `age`'s lack of normalization of `H:M:S` into the years, months and days field. However, this was also broken for values > 1 month, as well as breaking `a::timestamp(tz) - b::timestamp(tz)` operators. This has now been resolved. [#57956][#57956]
```
Comment (mikeCRL) [inline releases/v20.1.11.md:50]:
```suggestion
- In v20.1.8, we attempted to fix `age`'s lack of normalization of `H:M:S` into the years, months, and days fields. However, this was also broken for values greater than 1 month, as well as breaking `a::timestamp(tz) - b::timestamp(tz)` operators. This has now been resolved. [#57956][#57956]
```

Suggested a minor edit, but it's also unclear whether the 20.1.8 fix introduced the other two bugs or just failed to address them. When navigating to the [PR](https://github.com/cockroachdb/cockroach/pull/57956) to see if I could get clarification, I noticed it doesn't have a description or link to an issue. Should we be ensuring this is present when we prep the release notes?

File: releases/v20.1.11.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Fixed a panic in protobuf decoding. [#58861][#58861]
+- Fixed a bug that caused errors when accessing a tuple column (`tuple.column` syntax) of a tuples that could be statically determined to be null. [#58899][#58899]
+- Fixed an internal error involving string literals used as arrays. [#59066][#59066]
+- GC jobs now populate the `running_status` column for [`SHOW JOBS`](..v20.1/show-jobs.html). This bug has been present since v20.1. [#59138][#59138]
+- Fixed a bug in which some non-conflicting rows provided as input to an [`INSERT ... ON CONFLICT DO NOTHING`](..v20.1/insert.html) statement could be discarded, and not inserted. This could happen in cases where the table had one or more unique indexes in addition to the primary index, and some of the rows in the input conflicted with existing values in one or more unique index. This scenario could cause the rows that did not conflict to be erroneously discarded. This is now fixed. [#59172][#59172]
+
+### Contributors
+
+This release includes 24 merged PRs by 17 authors.
+We would like to thank the following contributors from the CockroachDB community:
```
Comment (mikeCRL) [inline releases/v20.1.11.md:65]: It is expected that the content below this line isn't showing up in the TeamCity preview?

File: releases/v20.1.11.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.11
+~~~
+
+
```
Comment (mikeCRL) [inline releases/v20.1.11.md:43]:
```suggestion
```

Dropping the extra blank line

File: releases/v20.1.11.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.11.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.11.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.11.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.11
```
Comment (mikeCRL) [inline releases/v20.1.11.md:40]: Per our chat, this needs updating from 'unstable'

**Review by taroface — COMMENTED** [review]
TFTR!!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9488#-:-MRvH86tAShqPtU_rp8O:bz5m8k5)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @mikeCRL)

___
*[releases/v20.1.11.md, line 40 at r1](https://reviewable.io/reviews/cockroachdb/docs/9488#-MRuY6mre5CwoocK8Fs2-r1-40:-MRvDazl0QpblAumwNaN:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/06c0c3cba62188a1570f60929a51a8f91e5641a3/releases/v20.1.11.md#L40)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

Per our chat, this needs updating from 'unstable'
</blockquote></details>

Done.
___
*[releases/v20.1.11.md, line 43 at r1](https://reviewable.io/reviews/cockroachdb/docs/9488#-MRuY6mre5CwoocK8Fs2-r1-43:-MRvDcq42lbA9s11JLO5:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/06c0c3cba62188a1570f60929a51a8f91e5641a3/releases/v20.1.11.md#L43)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
```

Dropping the extra blank line
</blockquote></details>

Done.
___
*[releases/v20.1.11.md, line 50 at r1](https://reviewable.io/reviews/cockroachdb/docs/9488#-MRuY6mre5CwoocK8Fs2-r1-50:-MRvGQlu59i3-X1EbsxW:bw1bbdo) ([raw file](https://github.com/cockroachdb/docs/blob/06c0c3cba62188a1570f60929a51a8f91e5641a3/releases/v20.1.11.md#L50)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
- In v20.1.8, we attempted to fix `age`'s lack of normalization of `H:M:S` into the years, months, and days fields. However, this was also broken for values greater than 1 month, as well as breaking `a::timestamp(tz) - b::timestamp(tz)` operators. This has now been resolved. [#57956][#57956]
```

Suggested a minor edit, but it's also unclear whether the 20.1.8 fix introduced the other two bugs or just failed to address them. When navigating to the [PR](https://github.com/cockroachdb/cockroach/pull/57956) to see if I could get clarification, I noticed it doesn't have a description or link to an issue. Should we be ensuring this is present when we prep the release notes?
</blockquote></details>

My read, based on the entry in the v20.1.8 release notes, is that the two bugs here weren't addressed by the initial fix. 

This commit in the linked PR contains the original release-note annotation: https://github.com/cockroachdb/cockroach/pull/57956/commits/59b2bc2186b49b866b6e417664d87530aae04f18

Since the script picked up this annotation, it should be in one of the commits!
___
*[releases/v20.1.11.md, line 65 at r1](https://reviewable.io/reviews/cockroachdb/docs/9488#-MRuY6mre5CwoocK8Fs2-r1-65:-MRvH2v62Gdclv_xbUUu:b-c06aa1) ([raw file](https://github.com/cockroachdb/docs/blob/06c0c3cba62188a1570f60929a51a8f91e5641a3/releases/v20.1.11.md#L65)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

It is expected that the content below this line isn't showing up in the TeamCity preview?
</blockquote></details>

Yes, this is just a way of defining links in Markdown!

**Pull request merged**